### PR TITLE
CORE-530: Add Network.confirmationsUntilFinal

### PR DESCRIPTION
**Pull request by EBGToo**
Pending Java.

### CODE REVIEW

File: crypto/BRCryptoNetwork.c [hunk 1]
```diff
@@ -303,6 +305,17 @@ cryptoNetworkSetHeight (BRCryptoNetwork network,
...
 }
 
+extern uint32_t
+cryptoNetworkGetConfirmationsUntilFinal (BRCryptoNetwork network) {
+    return network->confirmationsUntilFinal;
+}
+
+private_extern void
+cryptoNetworkSetConfirmationsUntilFinal (BRCryptoNetwork network,
+                                         uint32_t confirmationsUntilFinal) {
```
Comment (michael-brd) [inline crypto/BRCryptoNetwork.c]: Do we want this to be a uint64_t (agreed, no chance the value would ever need that magnitude) just to have all "height" values a uniform size?

File: crypto/BRCryptoNetwork.c [hunk 2]
```diff
@@ -303,6 +305,17 @@ cryptoNetworkSetHeight (BRCryptoNetwork network,
     network->height = height;
 }
 
+extern uint32_t
+cryptoNetworkGetConfirmationsUntilFinal (BRCryptoNetwork network) {
+    return network->confirmationsUntilFinal;
```
Comment (michael-brd) [inline crypto/BRCryptoNetwork.c]: TSAN will flag this.

**Comment by michael-brd** [comment]
Java ready to go, pending BDB working again ☹️

File: crypto/BRCryptoNetwork.c (hunk 1 above)
Comment (EBGToo) [reply crypto/BRCryptoNetwork.c]: See CORE-533.

File: crypto/BRCryptoNetwork.c (hunk 2 above)
Comment (EBGToo) [reply crypto/BRCryptoNetwork.c]: See CORE-523.

**Comment by michael-brd** [comment]
> Java ready to go, pending BDB working again ☹️

Java added; approved!

**Pull request merged**